docs/csharp/tutorials/intro-to-csharp/branches-and-loops-local.md

```diff
@@ -1,18 +1,18 @@
 ---
-title: Branches and loops tutorial - C# local quickstarts
-description: In this quickstart about branches and loops, you write C# code to explore the language syntax that supports conditional branches and loops to execute statements repeatedly.
+title: Branches and loops tutorial - C# introduction tutorial
```

Comment:
nit: duplication of the "tutorial" word and not consistent with the collections tutorial (which title ends with "Introduction to C#")

Reply:
Good catch. I changed all the titles to be **<subject> - Introduction to C# tutorial**, or **<subject> - Introduction to C# interactive tutorial**. The only exception is the string interpolation tutorial, which is not listed as "introductory".